MAINTAINERS.md

```diff
@@ -0,0 +1,38 @@
+# Maintainers
+
+This doc describes how to prep a release and publish to GitHub and `npm`.
+
+## Prep the Release
+
+Run the following command, where `<version>` is the version you wish to update the package to:
+
+```sh
+npm run bump -- <version>
+```
+
+This command will automatically bump the version in the `package.json` and the `package-lock.json` files, as well as add the corresponding tags.
+
+## Publishing to GitHub :octopus:
+
+The next step is to push these changes to GitHub:
+
+<!-- TODO: figure out the one `git push` command that can push both code and tags... but this is fine in the meantime? -->
```

Comment:
@erunion do you feel good about this `git push` guidance? I never push tags like this so not sure if there's a better way to do this

Reply:
Yeah this is fine, this is how I normally do it with my [`pushtags` alias](https://github.com/erunion/dotfiles/blob/main/gitconfig#L19). I'm not sure if there's a oneliner or not.